pnpm-workspaces/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "@tsconfig/node16-strictest/tsconfig.json",
+  "compileOnSave": false,
+  "compilerOptions": {
+    "noImplicitAny": false
```

Comment:
Is this required?

Reply:
Nope. Will remove.

Reply:
Yes, because I haven't typed anything in the local TS-libs (this allows implicit any).

Reply:
Removed `compileOnSave` tho.